packages/itwinui-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -5,24 +5,32 @@
 import * as React from 'react';
 import type { HeaderProps } from '../../../react-table/react-table.js';
 import { Checkbox } from '../../Checkbox/Checkbox.js';
-import { SvgColumnManager } from '../../../utils/index.js';
-import { DropdownMenu } from '../../DropdownMenu/DropdownMenu.js';
+import { SvgColumnManager, type PortalProps } from '../../../utils/index.js';
 import { IconButton } from '../../Buttons/IconButton.js';
-import { MenuItem } from '../../Menu/MenuItem.js';
 import { tableResizeStartAction } from '../Table.js';
 import { SELECTION_CELL_ID } from './selectionColumn.js';
 import { EXPANDER_CELL_ID } from './expanderColumn.js';
+import { Popover, usePopover } from '../../Popover/Popover.js';
+import { Surface } from '../../Surface/Surface.js';
+import { Flex } from '../../Flex/Flex.js';
 
 const ACTION_CELL_ID = 'iui-table-action';
 
 type ActionColumnProps = {
   columnManager?:
     | boolean
     | {
-        dropdownMenuProps: Omit<
-          React.ComponentPropsWithoutRef<typeof DropdownMenu>,
-          'menuItems' | 'children'
-        >;
+        dropdownMenuProps: React.ComponentPropsWithoutRef<'div'> & {
+          /**
+           * ARIA role. Role of menu. For menu use 'menu', for select use 'listbox'.
+           * @default 'menu'
+           */
+          role?: string;
+        } & Pick<
+            Parameters<typeof usePopover>[0],
+            'visible' | 'onVisibleChange' | 'placement' | 'matchWidth'
+          > &
+          Pick<PortalProps, 'portal'>;
```

Comment:
I think we can just use `Popover` props with some of them omitted.

```ts
dropdownMenuProps: Omit<
  React.ComponentPropsWithoutRef<typeof Popover>,
  'children' | 'content'
>;
```

Reply:
I'm mostly ok with this, except `matchWidth` does nothing on `Popover` and will show up in the DOM props if set.

Even previously, `matchWidth` was totally unnecessary, I'm not sure why were exposing it.

Reply:
> So, are you suggesting no longer exposing `matchWidth` in `ActionColumn`'s `dropdownMenuProps`?

Yes lets remove it.

> Also, when setting `matchWidth={true}`, I didn't see any difference ([playground](https://stackblitz.com/edit/github-few12s?file=src%2FApp.tsx)).

It only does something when the menu-items are narrower than the trigger element.

![](https://github.com/iTwin/iTwinUI/assets/9084735/7ea718be-b679-4f42-9c7e-78c24901d410)

Since this is an impossible case for column manager, we don't need this prop.